src/Search/Input/QueryBuilder.php

```diff
@@ -412,6 +412,7 @@ public static function displayCriteria($request = [])
             'criteria'   => $criteria ?? null,
             'prefix'     => $prefix,
             'p'         => $p,
+            'mainform'   => $p['mainform'] ?? 0,
```

Comment:
@cconard96 I'm not sure about this one.
There is a bug in list of users page where slider controls doesn't work anymore.
I found the mainform param was undefined when constructing first criteria.
Could you validate the fix is good ?